winrm/request.go

```diff
@@ -35,19 +34,20 @@ func NewOpenShellRequest(uri string, params *Parameters) (message *soap.SoapMess
 
 func NewDeleteShellRequest(uri string, shellId string, params *Parameters) (message *soap.SoapMessage) {
 	if params == nil {
-		params = DefaultParameters()
+		params = DefaultParameters
 	}
 	message = soap.NewMessage()
 	defaultHeaders(message, uri, params).Action("http://schemas.xmlsoap.org/ws/2004/09/transfer/Delete").ShellId(shellId).ResourceURI("http://schemas.microsoft.com/wbem/wsman/1/windows/shell/cmd").Build()
 
+	// TODO(masterzen) this func returns a new body and is not assigned to anything
```

Comment:
Yeah , my mistake ,i didn't know the method is a pointer receiver and it after the func modifies the struct.